launcher/launcher/main.go

```diff
@@ -45,6 +45,9 @@ var rcMessage = map[int]string{
 	holdRC:    "VM remains running",
 }
 
+// Version shows the commit version, set by -ldflags when building
+var Version = "dev"
```

Comment:
nit: call this `Commit` or `BuildCommit`

Reply:
Done